Source/Scene/InvertClassification.js

```diff
@@ -363,8 +329,7 @@ InvertClassification.prototype.isDestroyed = function () {
 };
 
 InvertClassification.prototype.destroy = function () {
-  this._fbo = this._fbo && this._fbo.destroy();
-  this._texture = this._texture && this._texture.destroy();
+  this._fbo.destroyResources();
```

Comment:
Does `fboClassified` need to be destroyed too?

Reply:
The classification texture isn't getting destroyed in `main` but I don't see anything weird if I add the extra `destroy` call..